convert_results.py

```diff
@@ -0,0 +1,182 @@
+"""This script converts JSON result files to Markdown summaries.
+
+The JSON format is meant to contain all details from tests. For human readable
+outputs, use this script to extract the most interest information from your JSON
+reports and write them into an eye-friendly markdown file.
+
+  Typical usage example:
+
+  python convert_results.py --source=results/ --target=result_summaries/
+"""
+
+import argparse
+import json
+import os
```

Comment:
Don't use os -> use  `from pathlib import Path`  this is the python3 way to manipulates files
https://docs.python.org/3/library/pathlib.html

Reply:
Done